app/assets/javascripts/turbo.js

```diff
@@ -4061,6 +4063,12 @@ class TurboCableStreamSourceElement extends HTMLElement {
     });
     return this.dispatchEvent(event);
   }
+  subscriptionConnected() {
+    this.setAttribute("connected", "");
```

Comment:
Would it make sense to add either add a timestamp or date to the attribute value, so that you could use that value to know when it was "[last] connected"?
```suggestion
    this.setAttribute("connected", new Date());
```
or
```suggestion
    this.setAttribute("connected", Number(new Date()));
```

Reply:
I'm inclined to leave it as-is, because I prefer the simplicity of a boolean attribute here.

That said, I also can't really think of a case where the timestamp would be needed. I'm only interested in the current state. Did you have a specific case in mind where you would need it?